webpages/config/db_name_sample.php

```diff
@@ -64,6 +64,8 @@
         // create users with a default password, but some external integrations to create users do so.  In that case, PlanZ can
         // identify users with this default password and prompt them to change it as well as report to staff. If your installation
         // does not use a default password, leave this empty ''.
+define("NEW_ACCOUNT_LINK_TIMEOUT", "PT48H"); // How long until new account link expires See https://www.php.net/manual/en/dateinterval.construct.php for format.
+define("NEW_ACCOUNT_LINK_TIMEOUT_DISPLAY", "2 days"); // Text description of NEW_ACCOUNT_LINK_TIMEOUT
```

Comment:
There's a trade-off here of usability against security. If we're auto-adding people who volunteer to be on programme, some number of those sign-ups are going to be at a convention, where they are less likely to be willing to go through the process to sign up for PlanZ immediately.